### PR TITLE
KIP7 solidity implementation

### PR DESCRIPTION
**Pull request by jimni1222**
## Proposed changes

This PR introduces implementation of KIP7.

The implementation of KIP7 included in this PR is based on openzeppelin's implementation of ERC20.

The difference between openzeppelin's ERC20 and its implementation is as follows:
- Remove the increaseAllowance / decreaseAllowance function.

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [ ] New feature or enhancement
- [x] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/caver-js/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/caver-js)
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

refer #220 
refer https://github.com/klaytn/caver-js/pull/222 

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: packages/caver-klay/caver-klay-kct/contract/token/KIP7/KIP7.sol [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+    function _mint(address account, uint256 amount) internal {
+        require(account != address(0), "KIP7: mint to the zero address");
+
+        _totalSupply = _totalSupply.add(amount);
+        _balances[account] = _balances[account].add(amount);
+        emit Transfer(address(0), account, amount);
+    }
+
+     /**
+     * @dev Destoys `amount` tokens from `account`, reducing the
```
Comment (kjhman21) [inline packages/caver-klay/caver-klay-kct/contract/token/KIP7/KIP7.sol]:
```suggestion
     * @dev Destroys `amount` tokens from `account`, reducing the
```

File: packages/caver-klay/caver-klay-kct/contract/token/KIP7/KIP7.sol [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+
+        _totalSupply = _totalSupply.sub(value);
+        _balances[account] = _balances[account].sub(value);
+        emit Transfer(account, address(0), value);
+    }
+
+    /**
+     * @dev Sets `amount` as the allowance of `spender` over the `owner`s tokens.
+     *
+     * This is internal function is equivalent to `approve`, and can be used to
```
Comment (kjhman21) [inline packages/caver-klay/caver-klay-kct/contract/token/KIP7/KIP7.sol]:
```suggestion
     * This internal function is equivalent to `approve`, and can be used to
```

File: packages/caver-klay/caver-klay-kct/contract/token/KIP7/KIP7.sol [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+    function _approve(address owner, address spender, uint256 value) internal {
+        require(owner != address(0), "KIP7: approve from the zero address");
+        require(spender != address(0), "KIP7: approve to the zero address");
+
+        _allowances[owner][spender] = value;
+        emit Approval(owner, spender, value);
+    }
+
+    /**
+     * @dev Destoys `amount` tokens from `account`.`amount` is then deducted
```
Comment (kjhman21) [inline packages/caver-klay/caver-klay-kct/contract/token/KIP7/KIP7.sol]:
```suggestion
     * @dev Destroys `amount` tokens from `account`.`amount` is then deducted
```

File: packages/caver-klay/caver-klay-kct/contract/token/KIP7/KIP7Burnable.sol [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+import "./KIP7.sol";
+
+/**
+ * @dev Extension of `KIP7` that allows token holders to destroy both their own
+ * tokens and those that they have an allowance for, in a way that can be
+ * recognized off-chain (via event analysis).
+ */
+contract KIP7Burnable is KIP7 {
+    /**
+     * @dev Destoys `amount` tokens from the caller.
```
Comment (kjhman21) [inline packages/caver-klay/caver-klay-kct/contract/token/KIP7/KIP7Burnable.sol]:
```suggestion
     * @dev Destroys `amount` tokens from the caller.
```

File: packages/caver-klay/caver-klay-kct/contract/README.md [hunk 5]
```diff
@@ -0,0 +1,4 @@
+caver-klay-kct/contract
+=================
+
+caver-klay-kct/contract refers to the contract implementataion in [openzeppelin](https://github.com/OpenZeppelin/openzeppelin-contracts).
```
Comment (kjhman21) [inline packages/caver-klay/caver-klay-kct/contract/README.md]:
```suggestion
Some files in `caver-klay-kct/contract` were derived from [openzeppelin](https://github.com/OpenZeppelin/openzeppelin-contracts).
```

File: packages/caver-klay/caver-klay-kct/contract/README.md [hunk 6]
```diff
@@ -0,0 +1,6 @@
+caver-klay-kct/contract
+=================
+
+Some files in `caver-klay-kct/contract` were derived from [openzeppelin](https://github.com/OpenZeppelin/openzeppelin-contracts).
```
Comment (KimKyungup) [inline packages/caver-klay/caver-klay-kct/contract/README.md]: Could you please write the version of openzeppelin code?

Comment (KimKyungup) [reply packages/caver-klay/caver-klay-kct/contract/README.md]: Refer this.
https://github.com/klaytn/klaytn/tree/dev/contracts/externals

Comment (jimni1222) [inline packages/caver-klay/caver-klay-kct/contract/README.md]:
```suggestion
Some files in `caver-klay-kct/contract` were derived from [openzeppelin contracts v2.5.0](https://github.com/OpenZeppelin/openzeppelin-contracts/releases/tag/v2.5.0).
```

File: packages/caver-klay/caver-klay-kct/contract/README.md [hunk 7]
```diff
@@ -0,0 +1,6 @@
+caver-klay-kct/contract
+=================
+
+Some files in `caver-klay-kct/contract` were derived from [openzeppelin contracts v2.5.0](https://github.com/OpenZeppelin/openzeppelin-contracts/releases/tag/v2.5.0).
```
Comment (jimni1222) [inline packages/caver-klay/caver-klay-kct/contract/README.md]:
```suggestion
Some files in `caver-klay-kct/contract` were derived from [openzeppelin contracts v2.5.0-rc.0](https://github.com/OpenZeppelin/openzeppelin-contracts/releases/tag/v2.5.0-rc.0).
```

File: packages/caver-klay/caver-klay-kct/contract/README.md [hunk 8]
```diff
@@ -0,0 +1,6 @@
+caver-klay-kct/contract
+=================
+
+Some files in `caver-klay-kct/contract` were derived from [openzeppelin contracts v2.5.0-rc.0](https://github.com/OpenZeppelin/openzeppelin-contracts/releases/tag/v2.5.0-rc.0).
```
Comment (jimni1222) [inline packages/caver-klay/caver-klay-kct/contract/README.md]:
```suggestion
Some files in `caver-klay-kct/contract` were derived from [openzeppelin contracts v2.3.0](https://github.com/OpenZeppelin/openzeppelin-contracts/releases/tag/v2.3.0).
```

File: packages/caver-klay/caver-klay-kct/contract/README.md (hunk 6 above)
Comment (jimni1222) [reply packages/caver-klay/caver-klay-kct/contract/README.md]: @KimKyungup Yes, I wrote the version together. :)

**Pull request merged**